packages/components/src/components/ConfirmationModal/ConfirmationModal.tsx

```diff
@@ -9,6 +9,7 @@ export interface ConfirmationModalProps {
   confirmLabel?: string;
   destructive?: boolean;
   onConfirm: () => void;
+  onCancel?: () => void;
```

Comment:
Why do you think we need a callback for this?

Reply:
To trigger actions (like track events, for example) when the user clicks on the cancel button

Reply:
I meant why changing this in the PR? Is there any specific scenario that would be necessary to override the state.hide action?

Reply:
The state.hide() will still be called here